### ISSUE TITLE
IP Banned

### ISSUE THREAD

**Issue by iPollox** [NoxxDev/NecroBot#409]
### Expected Behavior
Detect possibly IP Banned

### Actual Behavior
Keeps spamming no usable pokestops in my range

### Other Information
Version: 0.9.4
Windows Version: 10
What is affected (catch, transfer, evolve, recycle, snipe, powerup/levelup, other): ALL

I got IP Banned exactly after 24 hours and 4 minutes of having the bot running, which is fine its a throw away account that I don't care about.

![untitled](https://cloud.githubusercontent.com/assets/5094754/17848828/a1b31310-6823-11e6-9c3f-ae74fd45ae5a.png)

**Comment by VBQL** [comment]
Can you show us your config file?

**Comment by iPollox** [comment]
@VBQL http://pastebin.com/raw/RQMuHMJQ

**Comment by VBQL** [comment]
I would suggest you to change the geo coords to the defaults as I am not sure about the PokeStop frequency in HongKong. Change your speed to 9, switch FastSoftBan bypass to true (or wait for around 2 hours).

**Comment by VBQL** [comment]
If you want the Asia exclusive, I would suggest Tokyo, it definitly has a high concentration of PokeStops.

**Comment by ComputerApp1234** [comment]
Ive only seen ip bans if i used too many bots on the same computer with the same ip. But the other possible problem could be your delays.. If you send too many requests to the server in a specific time you get an ip ban. So i think you should change your delays to **"DelayBetweenPokemonCatch": 4000+** and **"DelayBetweenRecycleActions": true** !!

**Comment by samuraitruong** [comment]
you got ban because you reach lv 21 in 24h

**Comment by knumchoke** [comment]
I don't know about reaching lv21 in 24hours will get you ban.

I got this ban yesterday too.
In my country, I just restart my internet router to get a new IP. Remember not bot more than 1 account per IP.

If you got soft ban that make you unable to loot a pokestop. Login by your smartphone or boot strap and try logout then login again a few time. The ban should be lift.

**Comment by darkuru** [comment]
could you keep us updated if you get a ban? Read some parts that such a softbans had a good chance to end up in a perma-ban.

**Comment by mvp256** [comment]
@ComputerApp1234 You didn't get any soft-ban in the past 9 days? Neither perma-ban? Could you please share your configuration and the bot version you're running with? Thanks!

**Comment by iPollox** [comment]
I had turned off my computer until now (I was sleeping) and its working again. My configurations are stabilized at about 10K/Hour after it was done catching all the /new/ pokemon I havent seen on this account.

I know its an IP banned because it wasnt working with this /IP/ as soon as I switched to my phone's personal hotspot connection it worked again and when I went back to my normal router internet it wasnt working again, I also know this is an IP banned because at one point on the /original/ version of this bot (not the forked version) my internet was cut off so I had to use my hotspot to stay connected while I didn't have internet and I got IP banned for sniping and teleporting all over the place BUT with an iPhone if I turn on Airplane mode for 5~10 seconds the IP changes and my account would immediately work after that.

I believe I was flagged with an IP banned because I was 24 hours in a row ONLINE or connected to their services, and I guess they don't believe in a normal human to be more than 24 hours online without a break.
